packages/coding-standard/tests/Rules/PreferredSmartFileSystemOverNativeFileSystemFunctionCallRule/Fixture/PregMatchCalled.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Tests\Rules\PreferredSmartFileSystemOverNativeFileSystemFunctionCallRule\Fixture;
+
+final class PregMatchCalled
+{
+    public function run()
+    {
+        return preg_match('pattern', 'value');
```

Comment:
- e.g. `file_get_contents` → [`Symplify\SmartFileSystem\SmartFileSystem`, `readFile`]

Just make example for 1 method, I'll complete the rest in configuration of the rule

Reply:
implemented.